MANIFEST.in

```diff
@@ -1,29 +1,13 @@
-include README.rst
```

Comment:
These are now automatically included thanks to @jdavies-st's suggestion to add them to setup.cfg